RELEASE.md

```diff
@@ -146,23 +146,33 @@ example, any of the following should trigger a major version increment:
 
     1. Update the docs site with the new version number and release notes:
 
-        1. Update the `uswds` Node dependency to the new version, e.g.:
+        1. Create a branch off `develop` in which you change the `uswds`
+           dependency in `package.json` to the new version from npm, e.g.:
 
             ```sh
             cd path/to/web-design-standards-docs
             export VERSION=1.0.0
             git fetch origin
             git checkout -b release-${VERSION} origin/develop
-            npm install --save-dev uswds@${VERSION}
+            npm install --save uswds@${VERSION}
             ```
 
-        1. Update the `version` [variable in
-           _config.yml](https://github.com/18F/web-design-standards-docs/blob/master/_config.yml#L3).
-
```

Comment:
This is obsolete as of https://github.com/18F/web-design-standards-docs/pull/349.